steps/datadog-mobile-app-run-tests/1.0.0/step.yml

```diff
@@ -0,0 +1,113 @@
+title: datadog-mobile-app-run-tests
+summary: |
+  Step to run Datadog Synthetics tests on your CI
+description: |
+  With the `synthetics-test-automation-bitrise-step-run-tests` step you can run Synthetics tests during your Bitrise CI and ensure all your teams using Bitrise can benefit from Synthetic tests at every stage of the software lifecycle.
+website: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests
+source_code_url: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests
+support_url: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests/issues
+published_at: 2024-03-01T10:58:14.103768+01:00
+source:
+  git: https://github.com/DataDog/synthetics-test-automation-bitrise-step-run-tests.git
+  commit: 2239150885d7cce9cec0abdd4072cd51b01ecdc0
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: run-tests.sh
+inputs:
+- api_key: $DATADOG_API_KEY
+  opts:
+    description: The API key used to query the Datadog API.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Datadog API Key
+- app_key: $DATADOG_APP_KEY
+  opts:
+    description: The application key used to query the Datadog API.
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: Datadog APP Key
+- config_path: ""
+  opts:
+    description: The global JSON configuration used when launching tests.
+    title: Global JSON configuration
+- device_ids: ""
+  opts:
+    description: Override the mobile device(s) to run your mobile test.
+    title: Device IDs
+- fail_on_critical_errors: false
+  opts:
+    description: A boolean flag that fails the CI job if no tests were triggered,
+      or results could not be fetched from Datadog.
+    title: Fail on critical errors
+- fail_on_missing_tests: false
+  opts:
+    description: Fail if at least one specified test with a public ID (using `public_ids`
+      or listed in a test file) is missing in a run (for example, if it has been deleted
+      programmatically or on the Datadog site).
+    title: Fail on missing tests
+- fail_on_timeout: true
+  opts:
+    description: A boolean flag that fails the CI job if at least one test exceeds
+      the default test timeout.
+    title: Fail on timeout
+- files: ""
+  opts:
+    description: Glob pattern to detect Synthetic test configuration files.
+    title: Synthetic test config files
+- junit_report: ""
+  opts:
+    description: The filename for a JUnit report if you want to generate one.
+    title: JUnit Report
+- locations: ""
+  opts:
+    description: String of locations separated by semicolons to override the locations
+      where your tests run.
+    title: Locations
+- mobile_application_version: ""
+  opts:
+    description: Override the default mobile application version to test a different
+      version within Datadog.
+    title: Mobile Application Version
+- mobile_application_version_file_path: ""
```

Comment:
Could mention a good value to add here in the description:
`$BITRISE_IPA_PATH` and `$BITRISE_APK_PATH`

[xcode-archive Step Outputs](https://github.com/bitrise-steplib/steps-xcode-archive/blob/f7d8619ec543651ff9bd65eafcc69452f4158ea2/step.yml#L464)
[android-build Outputs](https://github.com/bitrise-steplib/bitrise-step-android-build/blob/bfebd365051f2b88761c8e3b06fda45cbe58e360/step.yml#L146)

Reply:
Thank you, this would be very useful for the users indeed, so i added it to the description 👍